### PR TITLE
Use services and modules packages from grafana/dskit

### PR DESCRIPTION
**Pull request by 56quarters**
Remove and replace the use of pkg/util/services and pkg/util/modules
with their equivalents from dskit. This is a mechanical change and no
behavior should be different.

Signed-off-by: Nick Pillitteri <nick.pillitteri@grafana.com>

**Checklist**
- [X] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: pkg/alertmanager/distributor_test.go [hunk 1]
```diff
@@ -13,13 +13,14 @@ import (
 	"testing"
 	"time"
 
+	"github.com/grafana/dskit/services"
+
```
Comment (pstibrany) [inline pkg/alertmanager/distributor_test.go:17]: extra newline

**Review by pstibrany — COMMENTED** [review]
We can also remove util/errors.go, util/module_service.go and util/modules subdirectory from Cortex now.

**Comment by 56quarters** [comment]
> We can also remove util/errors.go, util/module_service.go and util/modules subdirectory from Cortex now.

Removed `errors.go`. `module_service.go` was removed in the initial commit but git thinks it's been moved to `vendor`. The contents of the `util/modules` have been removed but the directory itself will [hang around](https://stackoverflow.com/questions/13590753/why-doesnt-git-version-directories) your local checkout until removed manually

**Review by pstibrany — APPROVED** [review]
> Removed `errors.go`. `module_service.go` was removed in the initial commit but git thinks it's been moved to `vendor`. The contents of the `util/modules` have been removed but the directory itself will [hang around](https://stackoverflow.com/questions/13590753/why-doesnt-git-version-directories) your local checkout until removed manually

You are right. I have misread the diff in my IDE, and haven't noticed the move properly.

Thanks, and also thank you for cleaning up imports in some files.

**Review by pracucci — APPROVED** [review]
LGTM, thanks!

**Pull request merged**